src/plot/hint.js

```diff
@@ -70,7 +70,7 @@ const ORIENTATION = {
  */
 function defaultFormat(value) {
   return Object.keys(value).map(function getProp(key) {
-    return {title: key, value: value[key]};
+    return { title: key, value: Object.prototype.toString.call(value[key]) === '[object Date]' ? value[key].toDateString() : value[key] };
```

Comment:
maybe break this out into a function and use typeof instead?

Reply:
I will look into it to change it. What ist your linting for the tests?

Reply:
I'm not sure I understand your question? We use eslint, you can run it locally by running 

```sh
npm run lint
```

Reply:
This throws 24237 erros:  Expected linebreaks to be 'LF' but found 'CRLF'  linebreak-style.
I thought that i have to change my linting to ignore those for example.

Reply:
Seems like Windows is the problem. I will work on it and try to resolve it.